README.md

```diff
@@ -1 +1,89 @@
-# art-institute-chicago-api
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Art Institute Chicago api
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved on external API.
+
```

Comment:
- [ ] Kindly update the project description to match this particular project instead of having Leaderboard description here.